### DISCUSSION TITLE
Sorting user_timeline by retweet_count?

### DISCUSSION THREAD

**Discussion by tanjyeyee** [tweepy/tweepy#1284] [Questions] [answered]
Is there a way to sort the tweets returned by user_timeline by retweet_count at the API level?

Since I'm going to use pagination, I need the tweets to be sorted before being returned to me.

**Comment by Harmon758** [comment, marked as answer]
No, this is not something that Twitter's API's [GET statuses/user_timeline endpoint](https://developer.twitter.com/en/docs/tweets/timelines/api-reference/get-statuses-user_timeline) supports.
